### PR TITLE
Block Number and Block Timestamp added during migration (truffle-reporters)

### PR DESCRIPTION
**Pull request by princesinha19**
I enhanced the code of truffle-reporters. Now, during truffle migration, we will also get block number and block timestamp. 
It is an enhancement. which could be useful for developers.

- [x] Tested the code.
- [x] Tested on Ganache and Geth

This PR solves: #1843

### CODE REVIEW

**Comment by princesinha19** [comment]
No problem, @CruzMolina. 
I will again look into #1843.

File: packages/truffle-reporters/reporters/migrations-V5/messages.js [hunk 1]
```diff
@@ -211,6 +211,14 @@ class MigrationsMessages {
             data.receipt.contractAddress
           }\n`;
 
+          output += `   > ${"Block Number:".padEnd(20)} ${
```
Comment (CruzMolina) [inline packages/truffle-reporters/reporters/migrations-V5/messages.js]:
```suggestion
          output += `   > ${"block number:".padEnd(20)} ${
```

File: packages/truffle-reporters/reporters/migrations-V5/messages.js [hunk 2]
```diff
@@ -211,6 +211,14 @@ class MigrationsMessages {
...
+            data.receipt.blockNumber
+          }\n`;
+
+          output += `   > ${"Block Timestamp:".padEnd(20)} ${
```
Comment (CruzMolina) [inline packages/truffle-reporters/reporters/migrations-V5/messages.js]:
```suggestion
          output += `   > ${"block timestamp:".padEnd(20)} ${
```

**Comment by princesinha19** [comment]
okay @CruzMolina. I will do the changes.

File: packages/truffle-reporters/reporters/migrations-V5/messages.js (hunk 1 above)
Comment (princesinha19) [reply packages/truffle-reporters/reporters/migrations-V5/messages.js]: I did this change.

**Pull request merged**